cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -71,6 +72,13 @@ define([
             );
         });
 
+        it('Changing course start date without active certificate configuration should result in error', function () {
+            this.view.$el.find('#course-start-date')
+                .val('10/06/2014')
+                .trigger('change');
+            expect(this.view.$el.find('span.message-error').text()).toContain("course must have atleast one active certificate configuration");
```

Comment:
if user-visible, "The course must have at least one active certificate configuration."